--- sftp-server.c~      2006-11-17 01:51:10.000000000 +0100
+++ sftp-server.c       2006-11-17 01:51:15.000000000 +0100
@@ -1217,7 +1217,7 @@
        extern char *__progname;
 
        /* Ensure that fds 0, 1 and 2 are open or directed to /dev/null */
-       sanitise_stdfd();
+       //sanitise_stdfd();
 
        __progname = ssh_get_progname(argv[0]);
        log_init(__progname, log_level, log_facility, log_stderr);
